elm.json

```diff
@@ -1,12 +1,13 @@
 {
     "type": "package",
-    "name": "cuducos/elm-format-number",
+    "name": "coinop-logan/elm-format-number",
     "summary": "Format numbers as pretty strings",
     "license": "BSD-3-Clause",
-    "version": "6.1.0",
+    "version": "1.0.0",
```

Comment:
We cannot downgrade according to Elm's versioning system. Would you mind reverting that -- you can leave the version control to the maintainers ; )